webui/react/src/pages/ModelDetails.tsx

```diff
@@ -70,8 +70,10 @@ const ModelDetails: React.FC = () => {
     NotLoaded: () => [],
   });
   const ensureWorkspacesFetched = useEnsureWorkspacesFetched(canceler.current);
-  const workspaces = Loadable.getOrElse([], useWorkspaces());
-  const workspace = workspaces.find((ws) => ws.id === model?.model.workspaceId);
+  const workspaces = useWorkspaces();
```

Comment:
nit: `lodableWorkspaces` is more accurate

Reply:
✔️